prepare.sh

```diff
@@ -3,4 +3,5 @@ if [[ "${PKGFORMAT-}" != "deb" && "${PKGFORMAT-}" != "rpm" ]]; then
   echo >&2 "This script requires PKGFORMAT to be either 'deb' or 'rpm'"
   exit 1
 fi
+set -x
```

Comment:
Is this intended to stay after development?

Reply:
This isn't a development option. The idea is that this script is run as part of our builds, and so far it's been invoked silently, which makes the build process less transparent. This command removes some of the magic from our builds.